src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[] | [];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  const neededComments = (comments.length !== 0)
+    ? (
+      <div className="CommentList">
+        {
+          comments
+            .map((comment: Comment) => (
+              <CommentInfo key={comment.id} comment={comment} />))
+        }
```

Comment:
```suggestion
        {comments
          .map((comment: Comment) => <CommentInfo key={comment.id} comment={comment} />)}
```

Reply:
if i write this in one line, i get more than 80 characters...